l5kit/l5kit/dataset/select_agents.py

```diff
@@ -231,7 +231,7 @@ def select_agents(
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument("--input_folders", nargs="+", type=str, required=True, help="zarr path")
-    parser.add_argument("--th_agent_prob", type=float, default=0.5, help="perception threshold on agents of interest")
+    parser.add_argument("--th_agent_prob", type=float, required=True, help="perception threshold on agents")
```

Comment:
Any reason for this?

Reply:
Because `th_agent_prob` is used for the folder name, it makes little sense to leave it with a default value. When you run this script that param is the one you want to move